### PR TITLE
'Storable' instance for KeyPoint data type

### PR DESCRIPTION
**Pull request by dmalkr**

### CODE REVIEW

File: opencv/src/OpenCV/Core/Types.hsc [hunk 1]
```diff
@@ -212,6 +216,14 @@ instance FromPtr KeyPoint where
 instance CSizeOf C'KeyPoint where
     cSizeOf _proxy = c'sizeof_KeyPoint
 
+
+instance Storable KeyPoint where
+    sizeOf _ = c'sizeof_KeyPoint
+    alignment _ = 8 -- ???
```
Comment (roelvandijk) [inline opencv/src/OpenCV/Core/Types.hsc]: Uhm.. This is tricky :-)

I just noticed that c++11 has an [alignof](http://en.cppreference.com/w/cpp/language/alignof) operator. We could use that to determine the proper alignment.

I'm thinking about another [HSC macro](https://github.com/LumiGuide/haskell-opencv/blob/master/opencv/include/hsc_macros.hpp) like `hsc_sizeof`.

Writing this `hsc_alignof` macro is mostly copy+pasting the `hsc_sizeof` macro.

Comment (roelvandijk) [inline opencv/src/OpenCV/Core/Types.hsc]: I added the macro: 884857725b02e8a930a2a4178d8e39e372585131

**Comment by dmalkr** [comment]
> Can you tell me a bit about the motivation of having a Storable instance for KeyPoint?

I need to save vector of KeyPoint to file, so I need Storable or open constructors for KeyPoint.
But now I bypass this and store KeyPointRec.

So this instance does not need in OpenCV project and can be removed.

**Comment by basvandijk** [comment]
IIUC this can be closed because we can serialize via `KeyPointRec`.

**Pull request closed without merging**